File-Formats/XlsIO/Working-with-Pivot-Tables.md

```diff
@@ -1,11 +1,11 @@
 ---
-title: Working with Pivot Tables | Excel library | Syncfusion
-description: In this section, you can learn how to create and use pivot tables operations in Excel document using XlsIO
+title: Working with Pivot Tables | Excel Library | Syncfusion
+description: Learn all about how to create and use pivot tables in Excel workbooks using Syncfussion’s essential Excel file-formats library for Windows Forms.
```

Comment:
Is this WinForms?
<img width="686" alt="image" src="https://github.com/syncfusion-content/fileformat-docs/assets/61148953/321e4c44-165d-416d-b439-2ed17dd75190">